content/sensu-core/0.29/reference/filters.md

```diff
@@ -245,26 +245,25 @@ attributes][8] (including custom attributes).
 ### Example filter attribute eval token
 
 The following is an example Sensu [filter definition][2], which is using a
-token (`:::check.occurrences|60:::`) as a secondary attribute in the Ruby eval
-expression. The token will be replaced by the [check definition attribute][6]
-named `occurrences` if it is defined, otherwise it will use the fallback value
-of `60`.
+token (`:::client.environent|known:::`) as a secondary attribute in the Ruby eval
```

Comment:
I think we want `:::client.environment|known:::` to be `:::client.environment|unknown:::` for tying into our example filter